data.sql

```diff
@@ -166,8 +166,15 @@ JOIN vets ON vets.id = visits.vets_id
 WHERE vets_id=2
 GROUP BY species.name LIMIT 1;
 
+-- week2  day1
 
 
```

Comment:
- [ ] This is a pair-programming paroject. But i cannot see any commit from your partner. Kindly ask your partner to add commit in this pull request :blush: